standards/API_STANDARDS.md

```diff
@@ -11,29 +11,40 @@ API standards
   * Search/list endpoints should contain consistent fields:
     * Response: `count`, `limit`, `offset`, `total_count`
 
-	  When pagination is required, the fields work as:
+	  When pagination is required, it is also required the data be in a sorted order before it can be paginated. The fields work as:
 
-	  `limit` - max number of items we're returning in this response (e.g. 20, or 50, rather than all we use atm)
+	  `limit` - max number of items we're returning in this response.
+	  * If value is not set, a configurable default limit should be used, unless otherwise stated this should be set to a value of `20`.
```

Comment:
At the moment, the default value is 100, we will need to change it if we decide to go with 20.

Reply:
The argument for lowering the default value from 100 to something like 20, is because this requires more resource server side when the API user may not want that many resources to be returned.

Now if the user wants more results returned in the items array then they can use the limit value to get more; so they don't lose out, they just have to set a limit value.

I left some ambiguity in this statement to allow us to use a different value if there is good cause, such as API user research says we should make it 30 not 20 because blah blah blah